Assets/Prefabs/ExitPortal.prefab

```diff
@@ -9715,6 +9716,22 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: e41431e3eba680b4998c1ff0624d4cb3, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
+--- !u!54 &5294384722759509742
+Rigidbody:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 1235273854538230}
+  serializedVersion: 2
+  m_Mass: 1000
+  m_Drag: 0
+  m_AngularDrag: 0
+  m_UseGravity: 0
+  m_IsKinematic: 0
```

Comment:
Why isn't the portal's rigidbody kinematic? Will it not work with the spring joint? If you can't make it kinematic, can you make the mass catastrophically large so nothing can ever move it?

Reply:
It has frozen constraints, so it can't be moved even on collision (if it's collider wasn't actually a trigger ofc). So, there's no concern of it every moving due to physics...

Reply:
oh ok perfect, I forgot about constraints